Python/Product/Analysis/ModuleAnalysis.cs

```diff
@@ -451,8 +451,8 @@ private class ErrorWalker : PythonWalker {
                     .Select(s => new MemberResult(s.Name, s.GetMergedAnalysisValues()))
                     .ToList();
             } catch (Exception) {
-                // TODO: log exception

```

Comment:
This TODO should stay here, though we probably need to let the exception propagate to be able to log it from outside Analysis.dll.